File: 2024/src/d00.test.ts
import { Solver } from "./d00";

const example = `foobarbaz`;

describe("d00 Solver", () => {
  const solver = new Solver(example);

  it("should solve part 1 examples", () => {
    expect(solver.part1()).toBe(0);
  });

  it("should solve part 2 examples", () => {
    expect(solver.part2()).toBe(0);
  });
});

File: 2018/test/aoc2018/d00_test.clj
(ns aoc2018.d00-test
  (:require [clojure.test :refer [deftest is]]
            [aoc2018.d00 :as d]))

(def example "foobarbaz")

(deftest part1-test
  (is (= 0 (d/part1 example))))

(deftest part2-test
  (is (= 0 (d/part2 example))))
